api/pkg/transformer/pipeline/create_table_op.go

```diff
@@ -17,7 +18,10 @@ func NewCreateTableOp(tableSpecs []*spec.Table) Op {
 	return &CreateTableOp{tableSpecs: tableSpecs}
 }
 
-func (c CreateTableOp) Execute(_ context.Context, env *Environment) error {
+func (c CreateTableOp) Execute(ctx context.Context, env *Environment) error {
+	span, _ := opentracing.StartSpanFromContext(ctx, "pipeline.CreateTableOp")
```

Comment:
We should still use the returned / updated `ctx` here.

Reply:
why? it is not passed to other function

Reply:
ah, it's okay to omit it then.